plugins/in_http/http_prot.c

```diff
@@ -81,6 +81,36 @@ static int sds_uri_decode(flb_sds_t s)
     return 0;
 }
 
+cfl_sds_t get_health_message()
+{
+    char time_str[128];
+    cfl_sds_t msg = NULL;
+    struct tm tm_time;
+    struct flb_time tm;
+
+    flb_time_get(&tm);
+    gmtime_r(&tm.tm.tv_sec, &tm_time);
+
+    snprintf(time_str, sizeof(time_str) - 1, "%04d-%02d-%02dT%02d:%02d:%02d.%09ldZ",
+             tm_time.tm_year + 1900, tm_time.tm_mon + 1, tm_time.tm_mday,
+             tm_time.tm_hour, tm_time.tm_min, tm_time.tm_sec, tm.tm.tv_nsec);
+
+    msg = cfl_sds_create_size(256);
+    if (!msg) {
+        return NULL;
+    }
+
+    cfl_sds_printf(&msg,
+                  "{\n"
+                  "  \"status\": \"ok\",\n"
+                  "  \"message\": \"I can only show you the door. You're the one that has to walk through it.\",\n"
```

Comment:
Why this message?

Maybe we can return the version information instead?

Reply:
Yes please, version info would be great

Reply:
> Why this message?
> 

The Matrix

> Maybe we can return the version information instead?

makes sense